charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/serviceaccount.yaml

```diff
@@ -3,11 +3,11 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+  name: {{ include "common.serviceAccount.name" . }}-apiservice
   annotations:
     "helm.sh/hook": pre-install,pre-upgrade,post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-weight": "-1"
```

Comment:
I'd add a comment here so we remember why we needed to add this. Possibly referencing https://github.com/helm/helm/pull/10787.

Reply:
done